build/change-version.js

```diff
@@ -64,6 +64,12 @@ async function main(args) {
     process.exit(1)
   }
 
+  [oldVersion, newVersion].forEach(arg => {
```

Comment:
Not 100%sure about this patch since it blindly replaces the string, but I don't think we want to introduce semver to check if the args are valid versions.

Reply:
I would move the 'v' replacement exactly after the initialization and maybe I would prefer `map` instead of `forEach`

Reply:
Not sure what you mean about `v`? As for map, I find this shorter, but feel free to push to this branch what you have in mind.

Reply:
line 67 to move to line 60
` [oldVersion, newVersion].map(arg => arg.startsWith('v')? arg.slice(1):arg)`